apidef/api_definitions.go

```diff
@@ -591,6 +527,75 @@ type RequestSigningMeta struct {
 	SignatureHeader string   `bson:"signature_header" json:"signature_header"`
 }
 
+// GraphQLConfig is the root config object for GraphQL middlewares
+type GraphQLConfig struct {
+	// Enabled indicates if GraphQL should be enabled
+	Enabled bool `bson:"enabled" json:"enabled"`
+	// GraphQLApi is the API specific configuration
+	GraphQLApi GraphQLApi `bson:"api" json:"api"`
```

Comment:
Better to change to `GraphQLAPI` with all references

Reply:
@furkansenharputlu what do you mean by "with all references"?

Reply:
In my understanding, it means to rename the type and field to `GraphQLAPI`, so it is consistent with the other definitions regarding naming.
I have changed it now.